### ISSUE TITLE
Mark PiP placement in video preview

### ISSUE THREAD

**Issue by lukas2511** [voc/voctomix#227]
It would be nice to have a rough outline on where a PiP of one input would be placed over another to e.g. make it easier to decide whether to show a slide-only take or to also embed the camera picture in the corner.

This would be something that kinda is solved by multi-step takes but I think it would be faster to just have this outline and being able to quickly tell if a slide would be fully visible with the camera picture in the corner.

I'm thinking of something like this (the red rectangle), just a lot lighter:

![mpv-shot0001](https://user-images.githubusercontent.com/85965/44885493-6cdeb380-acc1-11e8-960e-7f744d4b255e.jpg)

**Comment by fightling** [comment]
Good idea. I'm also working on a preview step where you can see the upcoming composite.

**Comment by ramdyne** [comment]
During 35c3 one of the mixers in Hall C was pointing to the laptop screen to show where the PiP might show up for so long that I added some gaffertape to mark the area for PiP. Other mixer angels really appreciated that (well, most of them).

I think having to compose the desired composition in preview before being able to see if it is safe to switch to PiP would perhaps take too long. Having the info available at a glance all the time (when the appropriate  preview/program is selected) would remove a few brain cycles here and there. 

I wouldn't suggest big red borders like above, but a 1-2 pixel wide line of negative pixels or something... Something subtle.

**Comment by fightling** [comment]
In voctomix 2 also custom cropping is an issue to display border lines. So that you're able to setup the camera to fit into the cropping before switching the composite. Are you with me about that @ramdyne ?

**Comment by ramdyne** [comment]
I think so, but can you perhaps explain it a little better? Your description is confusing me a little.... I'm really sorry about that :-)
